### ISSUE TITLE
Curl Issues

### ISSUE THREAD

**Issue by ThomasHolbrook** [igeekjsc/JSSAPIScripts#2]
On some systems it seems that when using a standard 'curl --silent -k --user' the script error's with -:1: parser error : Start tag expected, '<' not found, due to the output from curl not being valid XML, if the curl commands are updated to 'curl --silent -H "Accept: text/xml" -k --user' to force curl to include the header it seems to return correct formatted XML that then passes through without issue. 

Not sure if its an issue, or something relevant on our OS(10.11.2) / JSS combination (9.8.2) but shared regardless. 

Cheers, 

Tom

**Comment by Jason411** [comment]
I'm seeing the same issue on 10.10.5 with JSS 9.81.  When starting off with "Categories" my error is this:

-:1: parser error : Start tag expected, '<' not found
{"categories":[{"id":12,"name":"Certificates"},{"id":2,"name":"Drivers"},{"id":6
^

**Comment by str8outtajc** [comment]
All - apologies - I was not getting email notifications on your comments.  So I'm just now seeing this.  Looking at it now.

**Comment by str8outtajc** [comment]
What OS and version of OS is your server on?

**Comment by ThomasHolbrook** [comment]
Hey! We have seen this on all sorts of combinations, 10.10 & 10.11 against Ubuntu and Windows JSS servers the 'curl --silent -H "Accept: text/xml" -k --user' adjustment seems to resolve these issues in all instances so its not been a issue. Thanks for the great script!

**Comment by talkingmoose** [comment]
I'm seeing this issue too now.

```
Fetching XML data for policies ID's
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed
100   207  100   207    0     0   1103      0 --:--:-- --:--:-- --:--:--  1112
-:1: parser error : Start tag expected, '<' not found
{"policies":[{"id":2,"name":"Add Microsoft Office 2016 Dock Items"},{"id":4,"nam
^


Creating a plain list of policies ID's
```

Looks like two curl commands in the current script are missing the `-H "Accept: text/xml"` argument.

Changing line 213 to include this resolves the issue for me. Probably need to also change line 111.

**Comment by str8outtajc** [comment]
Fixed - Version 1.22
